back/engines/commercial/bulk_import_ideas/app/controllers/bulk_import_ideas/web_api/v1/import_ideas_controller.rb

```diff
@@ -71,31 +79,27 @@ def bulk_create_params
     def import_ideas_service
       locale = params[:import_ideas] ? bulk_create_params[:locale] : current_user.locale
       personal_data_enabled = params[:import_ideas] ? bulk_create_params[:personal_data] || false : false
-      @import_ideas_service ||= if import_scope == :phase
-        phase_id = params[:id]
-        ImportProjectIdeasService.new(current_user, @project.id, locale, phase_id, personal_data_enabled)
-      else
-        ImportGlobalIdeasService.new(current_user)
-      end
-    end
-
-    def import_scope
-      return :phase if request.path.include? 'phases'
-
-      :global
+      phase_id = params[:id]
+      @import_ideas_service = ImportProjectIdeasService.new(current_user, @project.id, locale, phase_id, personal_data_enabled)
```

Comment:
It looks like `ImportProjectIdeasService#initialize` does quite a few operations including database queries. Should we memoize it?
```suggestion
      @import_ideas_service ||= ImportProjectIdeasService.new(current_user, @project.id, locale, phase_id, personal_data_enabled)
```